WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormRemoteActionUseCaseTests.swift

```diff
@@ -100,12 +106,81 @@ final class ProductFormRemoteActionUseCaseTests: XCTestCase {
     }
 
     // MARK: - Editing a product (`addProduct`)
+    func test_editing_product_and_password_when_WooCommerce_version_is_below_8_1_returns_false() {
+        // Given
+        let plugin = SystemPlugin.fake().copy(siteID: siteID,
+                                              name: "WooCommerce",
+                                              version: "8.0.0",
+                                              active: true)
+        storageManager.insertSampleSystemPlugin(readOnlySystemPlugin: plugin)
+
+        let sut = ProductPasswordEligibilityUseCase(stores: storesManager, storageManager: storageManager)
+
+        // When
+        let isEligible: Bool = sut.isEligibleForWooProductPasswordEndpoint()
+
+        // Then
+        XCTAssertFalse(isEligible)
+    }
+
+    func test_editing_product_and_password_when_WooCommerce_version_is_above_8_1_returns_true() {
+        // Given
+        let plugin = SystemPlugin.fake().copy(siteID: siteID,
+                                              name: "WooCommerce",
+                                              version: "8.2.0",
+                                              active: true)
+        storageManager.insertSampleSystemPlugin(readOnlySystemPlugin: plugin)
 
-    func test_editing_product_and_password_without_edits_does_not_trigger_actions_and_returns_success_result() {
+        let sut = ProductPasswordEligibilityUseCase(stores: storesManager, storageManager: storageManager)
+
+        // When
+        let isEligible: Bool = sut.isEligibleForWooProductPasswordEndpoint()
```

Comment:
Same comment as above about the misplaced test.

Reply:
Updated with above commits 👍